src/context/clientSDKCacheContext.ts

```diff
@@ -1,17 +1,24 @@
-import { AssetList, IdEither } from '@cognite/sdk';
-import { TimeSeries } from '@cognite/sdk/dist/src/resources/classes/timeSeries';
+import {
+  AssetList,
+  CogniteClient,
+  IdEither,
+  TimeSeriesList,
+} from '@cognite/sdk';
+import { AssetsAPI } from '@cognite/sdk/dist/src/resources/assets/assetsApi';
+import { TimeSeriesAPI } from '@cognite/sdk/dist/src/resources/timeSeries/timeSeriesApi';
```

Comment:
this tights us with to current internal structure (which is not very good)
instead we could use
```
type AssetsApi = CogniteClient['assets']
type TimeseriesApi = CogniteClient['timeseries']
```
```

Reply:
agree, will redo